modules/tinymce/src/core/main/ts/api/Options.ts

```diff
@@ -904,6 +904,20 @@ const register = (editor: Editor): void => {
     default: true
   });
 
+  registerOption('current_user_id', {
+    processor: 'string',
+  });
+
+  registerOption('fetch_user_by_id', {
+    processor: 'function',
+    default: () => Promise.reject()
```

Comment:
Adda reject reason `Promise.reject(new Error('fetch_user_by_id option not defined')` or something.